challenges/desafio23.js

```diff
@@ -0,0 +1,28 @@
+db.voos.find(
+  {
+    $and: [
+      {litrosCombustivel: {$not: {$gt: 1000}}},
+      {litrosCombustivel: {$exists: true}}
+    ]
+  },
+  {
+    vooId: 1,
+    litrosCombustivel: 1,
+    _id: 0
+  }
+).limit(1);
```

Comment:
Outra maneira com o $and implícito e sem o limit(1):
```suggestion
db.voos.findOne({ litrosCombustivel: { $lte: 1000, $exists: true } }, { vooId: 1, litrosCombustivel: 1, _id: 0 });
```